byterun/sys.c

```diff
@@ -309,7 +309,9 @@ CAMLprim value caml_sys_getcwd(value unit)
 #ifdef HAS_GETCWD
   if (getcwd(buff, sizeof(buff)) == 0) caml_sys_error(NO_ARG);
 #else
-  if (getwd(buff) == 0) caml_sys_error(NO_ARG);
+  #ifdef HAS_GETWD
```

Comment:
I thought some C preprocessors might only accept directives when they start at column zero, but I don't know the details.